src/sagemaker/jumpstart/curated_hub/curated_hub.py

```diff
@@ -205,37 +217,12 @@ def _populate_latest_model_version(self, model: Dict[str, str]) -> Dict[str, str
         )
         return {"model_id": model["model_id"], "version": model_specs.version}
 
-    def _get_jumpstart_models_in_hub(self) -> List[Dict[str, Any]]:
-        """Returns list of `HubContent` that have been created from a JumpStart model."""
-        hub_models = self.list_models()
-
-        js_models_in_hub = []
-        for hub_model in hub_models["HubContentSummaries"]:
-            # TODO: extract both in one pass
-            jumpstart_model_id = next(
-                (
-                    tag
-                    for tag in hub_model["search_keywords"]
-                    if tag.startswith(JUMPSTART_HUB_MODEL_ID_TAG_PREFIX)
-                ),
-                None,
-            )
-            jumpstart_model_version = next(
-                (
-                    tag
-                    for tag in hub_model["search_keywords"]
-                    if tag.startswith(JUMPSTART_HUB_MODEL_VERSION_TAG_PREFIX)
-                ),
-                None,
-            )
-
-            if jumpstart_model_id and jumpstart_model_version:
-                js_models_in_hub.append(hub_model)
-
-        return js_models_in_hub
+    def _get_jumpstart_models_in_hub(self) -> List[HubContentSummary]:
+        hub_models = summary_list_from_list_api_response(self.list_models())
+        return [model for model in hub_models if get_jumpstart_model_and_version(model) is not None]
```

Comment:
why would they be `None`?

Reply:
and would we want to log a warning in that case?

Reply:
We'd need to use the `get_*` function in order to retrieve the values for other logic in `sync`, but I don't mind making another helper util that uses the `get_*` function internally